python/tutorials/03-matrix-multiplication.py

```diff
@@ -182,6 +182,14 @@
         triton.Config({'BLOCK_SIZE_M': 128, 'BLOCK_SIZE_N': 32, 'BLOCK_SIZE_K': 32, 'GROUP_SIZE_M': 8}, num_stages=1, num_warps=4),
         triton.Config({'BLOCK_SIZE_M': 64, 'BLOCK_SIZE_N': 32, 'BLOCK_SIZE_K': 32, 'GROUP_SIZE_M': 8}, num_stages=1, num_warps=2),
         triton.Config({'BLOCK_SIZE_M': 32, 'BLOCK_SIZE_N': 64, 'BLOCK_SIZE_K': 32, 'GROUP_SIZE_M': 8}, num_stages=1, num_warps=2),
+        triton.Config({'BLOCK_SIZE_M': 64, 'BLOCK_SIZE_N': 32, 'BLOCK_SIZE_K': 64, 'GROUP_SIZE_M': 4}, num_stages=1, num_warps=2),
```

Comment:
I'd suggest we do not add more configs in the tutorial, since we rely on the API in [PR309](https://github.com/ROCmSoftwarePlatform/triton/pull/309) to do tuning.

Reply:
I think we can just replace the `num_stages=1` with `num_stages=0` since it reports the best performance.

Reply:
Of course, keeping both does not hurt performance, but it take a little longer to run the test.